sql/src/main/java/io/crate/metadata/cluster/RenameTableClusterStateExecutor.java

```diff
@@ -220,4 +218,21 @@ private static void validateAllIndicesAreClosed(ClusterState currentState,
             }
         }
     }
+
+    private static void renameIndices(MetaData metaData,
```

Comment:
Is this the same as in `RenameIndexClusterStateExecutor` ?

Reply:
it's slightly different, it uses an `String[]` for sourceIndex instead of `Index[]`. 
We do not really need `Index` as we already have the concrete indices and also got rid of the array re-ordering (see previous commits)

Reply:
I've now changed it to operate on a `List<RenameIndexAction>`